src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import BookDisplay from './components/BookDisplay';
+import Nav from './components/NavBar';
+import BookCategories from './components/BookCategory';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Nav />
+      <Routes>
+        <Route path="bookstore" element={<BookDisplay />} />
```

Comment:
- [ ] The **BookDisplay** component here should be your root route. Kindly change this route path here to `'/'`.

  ```
  <Route path="/" element={<BookDisplay />} />
  ```

Reply:
change made